### ISSUE TITLE
support for vg format?

### ISSUE THREAD

**Issue by ekg** [smondet/plawireg#2]
It'd be great if this and vg could speak in protobuf streams!

**Comment by smondet** [comment]
I will keep that in mind indeed.

(this should be the format definition then: https://github.com/ekg/vg/blob/master/vg.proto)

**Comment by ekg** [comment]
Yep, this or alternatively use GFA (an equivalent text format which it reads and writes).
